packages/blade/src/components/ActionList/actionListUtils.ts

```diff
@@ -77,7 +77,20 @@ const getActionListProperties = (
       actionListOptions.push({
         title: child.props.title,
         value: child.props.value,
-        href: child.props.href,
+        onClickTrigger: (value) => {
+          const anchorLink = child.props.href;
+          child.props.onClick?.({
+            name: child.props.value,
+            value: child.props.isSelected ?? value,
+          });
+
+          if (anchorLink) {
+            window.location.href = anchorLink;
```

Comment:
Won't this do a full page reload? what if users wants to do client side navigation?

Reply:
Kind of unsure what our client-side navigation strategy is right now also. Also, this only gets triggered when you select item with keyboard. Otherwise it will just work like a normal link.

I think once we we resolve https://github.com/razorpay/blade/issues/979, it will be clearer on what to do here and in other links. Kind of applies to `<ListItemLink />` kind of components as well since we can't use ReactRouter there because of validations. 

Added a comment here to keep track of this - https://github.com/razorpay/blade/issues/979#issuecomment-1550510357